docs/source/whatsnew/1.1.1.txt

```diff
@@ -1,23 +1,49 @@
-Development
------------
+Release 1.1.1
+-------------
 
 :Release: 1.1.1
-:Date: TBD
-
-.. warning::
-   This release is still under active development.  All changes listed are
-   subject to change at any time.
+:Date: June 30, 2017
 
 
 Highlights
 ~~~~~~~~~~
 
-None
+Since the the release of Zipline ``1.1.0``, Quantopian has officially released support for Futures in Zipline, as well as Python 3.5 support.
```

Comment:
What do you think about "Zipline now has broad support for futures, in addition to equities. It's also being tested and packaged for Python 3.5.", ignoring official-ness or the version?